src/NuGet.Core/NuGet.Packaging/PackageCreation/Resources/NuGetResources.resx

```diff
@@ -186,4 +186,7 @@
   <data name="Manifest_ReferencesIsEmpty" xml:space="preserve">
     <value>The element package\metadata\references\group must contain at least one &lt;reference&gt; child element.</value>
   </data>
+  <data name="SemVer2VersionsNotSupported" xml:space="preserve">
+    <value>Versions using SemVer 2.0.0.0 are not supported: {0}.</value>
```

Comment:
nit: the extra `.0` at the end of the SemVer version is not standard. See heading:
http://semver.org/